crates/biome_json_analyze/tests/specs/nursery/noDuplicateDependencies/invalid/package.json.snap

```diff
@@ -0,0 +1,42 @@
+---
+source: crates/biome_json_analyze/tests/spec_tests.rs
+expression: package.json
+---
+# Input
+```json
+{
+  "dependencies": {
+    "foo": "",
+    "bar": ""
+  },
+  "devDependencies": {
+    "foo": ""
+  }
+}
+
+```
+
+# Diagnostics
+```
+package.json:3:5 lint/nursery/noDuplicateDependencies ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i The dependency foo is also listed later.
```

Comment:
To be more helpful, it would be good if the diagnostic pointed out in which section the dependency also occurs.

```suggestion
  i The dependency foo is also listed under devDependencies.
```

Reply:
Makes sense, yes!

Reply:
https://github.com/biomejs/biome/pull/7142/commits/785c203213394f8de5c73720f1a6548e72613d11 :)